frontend/src/campaign/campaignView.js

```diff
@@ -214,6 +216,62 @@ Results.propTypes = {
     countryName: PropTypes.string,
 };
 
+class Citizen extends React.Component {
```

Comment:
Is this copy/paste from the budget sim? For the sake of expediency right now, I'm okay with that for now, but we should properly factor this out later.

Reply:
Yeah I can factor this out when we use it for feedback later

Reply:
sounds good!